src/com/pentalog/pentastagiu/homework2_bookCandyShop/Product.java

```diff
@@ -0,0 +1,63 @@
+package com.pentalog.pentastagiu.homework2_bookCandyShop;
+
+import java.util.Objects;
+
+public class Product {
+
+    private String productId;
+    private String productName;
+    private Double price;
+
+    //constructors
+    public Product() {
+    }
+
+    public Product(String productId, String productName, Double price) {
+        this.productId = productId;
```

Comment:
atat aici cat si in subclase, trebuie sa faci niste validari: in constructori si in metode publice(setteri in special) trebuie sa te asiguri ca nu sunt introduse date invalide: price si quantity negative de exemplu

totodata ti-as recomanda sa incepi cu fielduri private iar setteri/getteri sa adaugi pe masura ce ai nevoie; cateodata char e nevoie doar de getteri, nu e nevoie sa generam toate metodele de get si set daca nu avem nevoie si asa scapam si de validarile pe care ar trebui sa le facem pe setteri